assets/test/protractor-chrome.conf.js

```diff
@@ -101,7 +101,7 @@ exports.config = {
   // testing on multiple browsers, use the multiCapabilities array.
 
   // For a list of available capabilities, see
-  // https://code.google.com/p/selenium/wiki/DesiredCapabilities
+  // https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities
```

Comment:
I had troubles compiling OS because of code.google.com repos migrating to github and other places so I updated affected imports in the [first commit](https://github.com/miminar/origin/commit/57df7ce2634c3eb5a78951e3542f967d6d3b4403). I guess it should be a separate PR.